x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/attach_index_box.tsx

```diff
@@ -144,7 +187,7 @@ export const AttachIndexBox: React.FC<AttachIndexBoxProps> = ({ connector }) =>
   }, [query]);
 
   useEffect(() => {
-    setSanitizedName(formatApiName(connector.name));
+    setSanitizedName(prefixConnectorIndex(formatApiName(connector.name)));
```

Comment:
For suggested connector name, if it's native connector we have to prepend `content-` (this is the big blue button if index is not attached yet)